packages/react/src/Accordion/Accordion.tsx

```diff
@@ -12,6 +12,10 @@ import { HeadingLevel } from '../Heading/Heading'
 import { useKeyboardFocus } from '../common/useKeyboardFocus'
 
 export type AccordionProps = {
+  /**
+   * The hierarchical level of the accordion title within the document.
+   * @default 1
```

Comment:
Can we document that we need to specify default values for components that don’t define them but just pass the prop on?

Reply:
Hm, this is actually an error. I copied this from Alert, but the Accordion headingLevel doesn't have a default. 

For the doc gen we don't need to define these defaults, it gets it from the default props. It does show in your IDE, but documenting that may be a bit out of scope for this PR?